tensorflow/python/eager/pywrap_tfe_src.cc

```diff
@@ -1799,6 +1800,7 @@ bool OpDoesntRequireInput(const string& op_name) {
           "BiasAdd",
           "Relu",
           "Relu6",
+          "LeakyRelu",
```

Comment:
The LeakyRelu gradient does seem to use the inputs? Same above.

Reply:
My fault, 'LeakyRelu' and 'LeakyReluGrad' both use the input and don't use the output, corrected in the new commit.